src/services/utilServices/SitesService.js

```diff
@@ -81,7 +102,16 @@ class SitesService {
         )
     })
 
-    return _.flatten(sites)
+    const flattenedAllSites = _.flatten(allSites)
+    // Github users are using their own access token, which already filters sites to only those they have write access to
+    if (isAdminUser || !isEmailUser) return flattenedAllSites
```

Comment:
sanity check here - should this be `!isAdminUser || !isEmailUser)` or simply `!isEmailUser`? from the comment, this seems like it should be a non-admin user using their own github access token (which follows the same flow as an admin user)

Reply:
Hmm the check is correct - to clarify, the flows are:

1. Email user - uses our centralised access token, so flattenedAllSites returns all sites, so we need to further restrict based on our database
2. Email user (admin) - uses our centralised access token, so flattenedAllSites returns all sites.
3. Github user - uses their own access token, so flattenedAllSites returns only those they have write access to
4. Github user (admin) - uses their own access token, so flattenedAllSites returns only those they have write access to (which is all sites)

I've added an extra comment about admin users in f23cbd2!

Reply:
gotcha i think i misunderstood the conditional as github login is caught by the `!isEmailUser` portion